nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -280,6 +284,43 @@ let
           '';
         };
 
+        podman = mkOption {
+          type = types.nullOr (
+            types.submodule {
+              options = {
+                sdnotify = mkOption {
+                  default = "conmon";
+                  type = types.enum [
+                    "conmon"
+                    "healthy"
+                    "container"
```

Comment:
Note: this probably requires some testing before merging. I haven't really verified if all the settings are correct.